invenio_cli/cli.py

```diff
@@ -122,14 +122,16 @@ def demo(local):
 
 
 @cli.command()
-def run():
+@click.option('--stop', default=False, is_flag=True,
+              help='Stop containers')
+def run(stop):
```

Comment:
After discussing IRL, the stop should be a separate *command*, not a flag on the *run*. (We we called option b when we talked)

```suggestion
def run():
```